src/app/modules/entities/entities-routing.module.ts

```diff
@@ -0,0 +1,31 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { Routes, RouterModule } from '@angular/router';
+import { StateEntityListComponent } from './state-entity-list/state-entity-list.component';
+import { ViewEntityDetailsComponent } from './view-entity-details/view-entity-details.component';
+
+const routes: Routes = [
+  {
+    path: '',
+    // data: { title: 'Home' },
+    children: [
+      {
+        path: 'list', component: StateEntityListComponent, data: {
+          title: [{ name: 'Home >', link: '/home' },
```

Comment:
we discussed to remove the arrow and put it inside the breadcrumb component

Reply:
Changed